Cargo.toml

```diff
@@ -11,7 +11,7 @@ path = "src/main.rs"
 anyhow = "1.0.75"
 derive_setters = "0.1.6"
 thiserror = "1.0.49"
-http-cache-semantics = { version = "1.0.1", default-features = false, features = [
+http-cache-semantics = { version = "1.0.0", default-features = false, features = [
```

Comment:
why the downgrade?

Reply:
@tusharmath The build was failing for some reason with v1.0.1 some borrowing error in `http-cache-semantics`

Reply:
can you revert this and try again? It's not failing on main.